src/frontend/src/pages/PIADetailPage/index.tsx

```diff
@@ -0,0 +1,172 @@
+import {
+  IPIAIntake,
+  IPIAIntakeResponse,
+} from '../../types/interfaces/pia-intake.interface';
+import { faArrowDown } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { dateToString } from '../../utils/date';
+import messages from './messages';
+
+import { useNavigate, useParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { HttpRequest } from '../../utils/http-request.util';
+import { API_ROUTES } from '../../constant/apiRoutes';
+import { routes } from '../../constant/routes';
+
+const PIADetailPage = () => {
+  // https://github.com/microsoft/TypeScript/issues/48949
+  // workaround
+  const win: Window = window;
+  const { id } = useParams();
+  const navigate = useNavigate();
+  const [pia, setPia] = useState<any>({});
+  useEffect(() => {
+    (async () => {
+      try {
+        // Actually perform fetch
+        const result = (
+          await HttpRequest.get<IPIAIntakeResponse>(
+            API_ROUTES.GET_PIA_INTAKE.replace(':id', `54`),
+          )
+        ).data;
+        setPia(result);
+      } catch (e) {
+        if (e instanceof Error && e.cause instanceof Error) {
+          const errorCode = e.cause.message as string;
+          if (errorCode.includes('404')) {
```

Comment:
I have added a suggestion below where `cause` is directly status code received, and not an error itself.

The reason I initially suggested this change so we could get rid of string comparisons; and remove `.includes` checks.

Reply:
i get rid of the includes compare and use === instead

Reply:
if we can't simply have `any` type for cause, that's fine. This still will have dependency to the message; But that is fine.